src/test/java/org/mule/extensions/internal/execution/JavaArgumentTransformationTestCase.java

```diff
@@ -152,33 +152,24 @@ public void invokeStaticWithMapWithListValuesAndOneIsNull() throws Exception {
   }
 
   @Test
-  public void invokeStaticWitNullParameterForPrimitiveValue() throws Exception {
+  public void invokeStaticWithNullParameterForPrimitiveValue() throws Exception {
     expectedError.expectError("JAVA", "ARGUMENTS_MISMATCH", ArgumentMismatchModuleException.class,
-                              "Failed to invoke Method [getSameNumber(int)] in Class " +
-                                  "[org.mule.extensions.internal.model.AnotherExecutableElement] with arguments [Object number]."
-                                  + " Expected arguments are [int number]: null");
-    TypedValue<String> payload = flowRunner("invokeStaticWitNullParameterForPrimitiveValue")
-        .run()
-        .getMessage()
-        .getPayload();
+                              "Failed to invoke static Method 'getSameNumber(int)' "
+                                  + "in Class 'org.mule.extensions.internal.model.AnotherExecutableElement'. "
```

Comment:
`in Class` or `from Class`? 🤔

Reply:
had the same internal debate, which one sounds better to you?